docs/code_tour.md

```diff
@@ -2,8 +2,37 @@
 This document is intended to provide a development guideline for new and existing contributors to the project.
 Keep in mind this application undergoes constant development so be sure to check back occasionally as best practices are sure to evolve.
 
-## Linting
-This project uses **ESLint** and **Sass Lint**. Both can be run with the script `yarn vet`.
+## Yarn Tasks 
+#### `yarn build`
+- Bundles SUI assets and places then in `../manageiq/public/ui/service/`.
```

Comment:
You will need an empty line after each bulleted list for the markdown parser to format this correctly.